setup.py

```diff
@@ -30,7 +31,7 @@
     # There are some restrictions on what makes a valid project name
     # specification here:
     # https://packaging.python.org/specifications/core-metadata/#name
-    name="mvs_eland",  # Required
+    name="mvs_tool",  # Required
```

Comment:
Is this how it would appear in the terminal for installation? Like so:
    
    pip install mvs_tool

Is it possible to change this name later? Not sure if "eland" has to be in it.

Reply:
It will be possible to change the name :)